explorer/explorertypes.go

```diff
@@ -134,21 +134,23 @@ type BlockInfo struct {
 
 // AddressInfo models data for display on the address page
 type AddressInfo struct {
-	Address          string
-	Limit            int64
-	Offset           int64
-	Transactions     []*AddressTx
-	NumFundingTxns   int64
-	NumSpendingTxns  int64
-	NumTransactions  int64
-	KnownFundingTxns int64
-	NumUnconfirmed   int64
-	TotalReceived    dcrutil.Amount
-	TotalSent        dcrutil.Amount
-	Unspent          dcrutil.Amount
-	Balance          *AddressBalance
-	Path             string
-	Fullmode         bool
+	Address           string
+	Limit             int64
+	MaxLimit          int64
+	Offset            int64
+	Transactions      []*AddressTx
+	NumFundingTxns    int64
```

Comment:
I take a lot of blame for ambiguous naming in this section, but I think we could use some comments on these counts.  e.g. 

        NumFundingTxns    int64  // The number of transactions paying to the address

down to `NumUnconfirmed` please